src/backend/nodes/cypher_readfuncs.c

```diff
@@ -186,8 +186,8 @@ void read_cypher_create_target_nodes(struct ExtensibleNode *node)
     READ_LOCALS(cypher_create_target_nodes);
 
     READ_NODE_FIELD(paths);
-    READ_UINT_FIELD(flags);
-    READ_UINT_FIELD(graph_oid);
+    READ_INT_FIELD(flags);
+    READ_INT_FIELD(graph_oid);
```

Comment:
Why were these changed back to INT from UINT. They were purposely changed to UINT in the master.